planetiler-core/src/main/java/com/onthegomap/planetiler/reader/osm/PbfDecoder.java

```diff
@@ -74,6 +78,20 @@ private static byte[] readBlobContent(Fileformat.Blob blob) {
         throw new FileFormatException("PBF blob contains incomplete compressed data.");
       }
       inflater.end();
+    } else if (blob.hasLz4Data()) {
```

Comment:
Thanks for adding this! Would you mind adding a test case in `PlanetilerTests` that verifies end-to-end reading from an lz4 pbf file? We have an existing test file `planetiler-core/src/test/resources/monaco-latest.osm.pbf` - if you could just convert that to lz4 and update `testPlanetilerRunner` (or add a similar test) to use it then that should cover it, thanks!